Polar/README.md

```diff
@@ -1,3 +1,73 @@
 # Polar Datacube Generation
 
-Information dedicated to the generation of the Polar Datacube.
\ No newline at end of file
+Information dedicated to the generation of the Polar Datacube.
+
+## Cube generation process
+
+The cube generation process is divided in four phases:
+
+### 1. Downloading required raw datasets
+
+Datasets were downloaded via the source information provided in `cube.geojson`. Datasets were downloaded in different formats: `.nc`, `.tif`, `.shp`. Some datasets were compressed and this step includes the extraction of the data.
+
+To download the datasets it is required to run the following file:
+
+```
+# Download all files
+inputs-collect/download-sources.py
+```
+
+### 2. Preprocessing datasets
```

Comment:
I am a bit surprised that I need to execute multiple python scripts for preprocessing the input datasets in some cases. Could this be facilitated?

Reply:
In some cases it was required to first convert them into zarr files. Everytime there is an export or a save there is a new script. I will put every python command for running the files into a bash file, so users don't have to run every single one.